openapi/p4pa-debt-position.openapi.yaml

```
@@ -19,6 +19,12 @@ paths:
           required: false
           schema:
             type: boolean
+        - name: generateIuv
```

@robertopasetti  what do you think if the parameter would be named pagopaPayment instead?
ASIS in myPay it was flagGenerateIuv
```suggestion
        - name: pagopaPayment
```